### PR TITLE
fix(virtualRepeat): Recover from scroll events that occur when hidden

### LINKED ISSUE #5448: md-autocomplete has a bug on demo site (in master version and 1.0.0rc2
Hello,

md-autocomplete has a bug on the demo site.
The steps to repeat:
1. Left click on auto-complete box to open it
2. Scroll to the very bottom
3. Select "Wisconsin"
4. Click on the "X" to clear the selction
5. An Empty menu will show up with no items rendered.

screen shot 2015-10-29 at 9 49 53 pm

### PR DESCRIPTION
**Pull request by kseamon**
Fixes #5448.

### CODE REVIEW

**Comment by kseamon** [comment]
@robertmesserle @jelbourn @ThomasBurleson

File: src/components/virtualRepeat/virtual-repeater.js [hunk 1]
```diff
@@ -197,6 +197,7 @@ VirtualRepeatContainerController.prototype.updateSize = function() {
       ? this.$element[0].clientWidth
       : this.$element[0].clientHeight;
 
+  this.handleScroll_();
```
Comment (jelbourn) [inline src/components/virtualRepeat/virtual-repeater.js]: Maybe add a comment as to why you're calling the scroll handler here?

Comment (kseamon) [inline src/components/virtualRepeat/virtual-repeater.js]: Done

**Comment by jelbourn** [comment]
LGTM

**Pull request closed without merging**